contracts/lib/RLPReader.sol

```diff
@@ -1,7 +1,10 @@
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
This should be Apache-2.0. https://github.com/hamdiallam/Solidity-RLP/blob/master/contracts/RLPReader.sol